### DISCUSSION TITLE
External Bluetooth controller

### DISCUSSION THREAD

**Discussion by CyclistHomeCoder** [icanos/hassio-plejd#228] [Q&A]
Hi,

is anyone aware of a way to utilize an external Bluetooth controller with this component?

Background:

I have my Home Assistant running in a virtualized environment with no access to a local Bluetooth adapter. Instead I was looking to utilize an external Bluetooth adapter in a Raspberry Pi.

Any ideas?

**Comment by oyvindwe** [comment]
Take a look at Bluetooth Proxy: https://esphome.github.io/bluetooth-proxies/

I don't know how to use it with this component though.
